README.md

```diff
@@ -34,8 +34,10 @@ UI components are located in [src/www/ui_components](src/www/ui_components). The
 
 Additional requirements for Android:
```

Comment:
No to address in this PR, but we have this Docker image that was intended to make build easier, but it's probably broken at this point: https://github.com/Jigsaw-Code/outline-client/blob/master/tools/build/Dockerfile

Reply:
Will try later.